cmd/root_options.go

```diff
@@ -20,7 +21,7 @@ type OutputOptions struct {
 type RootOptions struct {
 	Architecture     string   `validate:"required,oneof=amd64 arm64" name:"architecture"`
 	DriverVersion    string   `default:"master" validate:"eq=master|sha1|semver" name:"driver version"`
-	KernelVersion    uint16   `default:"1" validate:"omitempty,number" name:"kernel version"`
+	KernelVersion    string   `default:"1" validate:"required" name:"kernel version"`
```

Comment:
Will changing this from `omitempty` to `required` break existing users that aren't supplying a value? Also, should we enforce `ascii` encoding as is done for `KernelRelease`?

Reply:
Good catch, let me set it back to `omitempty`. I had changed that very early on my development on the branch thinking I might have to go down a different path.